.circleci/config.yml

```diff
@@ -41,8 +41,11 @@ jobs:
       - run: PYTHON_VERSION=3.6 GIT_VERSION=1.8.0  bash ci/tox/ci-run.sh  # Earliest versions of python/git supported by git-machete
       - run: PYTHON_VERSION=3.7 GIT_VERSION=2.7.6  bash ci/tox/ci-run.sh
       - run: PYTHON_VERSION=3.8 GIT_VERSION=2.25.0 bash ci/tox/ci-run.sh
-      - run: PYTHON_VERSION=3.9 GIT_VERSION=2.30.1 bash ci/tox/ci-run.sh
+      - run: PYTHON_VERSION=3.9 GIT_VERSION=2.30.1 CHECK_COVERAGE="true" bash ci/tox/ci-run.sh
```

Comment:
Nit: quotes around `true` aren't necessary, the effect (i.e. the exact bytes passed in the env vars to the `bash` process) will be the same where you use `"`, `'` or no quotes.

This won't be so easy tho if the string wasn't `true` but contained spaces or other special characters which would be interpreted by bash, like `$` or `(`

Reply:
Rel: https://virtuslab.github.io/tips/#shell/input-output-redirections point 4